roles/flannel/README.md

```diff
@@ -13,15 +13,15 @@ to 0.3.
 Role Variables
 --------------
 
-| Name                | Default value                           | Description                                   |
-|---------------------|-----------------------------------------|-----------------------------------------------|
-| flannel_interface   | ansible_default_ipv4.interface          | interface to use for inter-host communication |
-| flannel_etcd_key    | /openshift.com/network                  | etcd prefix                                   |
-| etcd_hosts          | etcd_urls                               | a list of etcd endpoints                      |
-| etcd_conf_dir       | {{ openshift.common.config_base }}/node | SSL certificates directory                    |
-| etcd_peer_ca_file   | {{ etcd_conf_dir }}/ca.crt              | SSL CA to use for etcd                        |
-| etcd_peer_cert_file | Openshift SSL cert                      | SSL cert to use for etcd                      |
-| etcd_peer_key_file  | Openshift SSL key                       | SSL key to use for etcd                       |
+| Name                 | Default value                           | Description                                   |
+|----------------------|-----------------------------------------|-----------------------------------------------|
+| flannel_interface    | ansible_default_ipv4.interface          | interface to use for inter-host communication |
+| flannel_etcd_key     | /openshift.com/network                  | etcd prefix                                   |
+| etcd_hosts           | etcd_urls                               | a list of etcd endpoints                      |
+| etcd_cert_config_dir | {{ openshift.common.config_base }}/node | SSL certificates directory                    |
```

Comment:
It's used by the `etcd_client_certificates` role / family of roles (dep of this role).